applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -230,25 +281,53 @@ Image transfer over SMP
 
 To perform DFU using the `nRF Connect Device Manager`_ mobile app, complete the following steps:
 
-.. include:: /device_guides/nrf52/fota_update.rst
-   :start-after: fota_upgrades_over_ble_nrfcdm_common_dfu_steps_start
-   :end-before: fota_upgrades_over_ble_nrfcdm_common_dfu_steps_end
+.. tabs::
 
-.. include:: /device_guides/nrf52/fota_update.rst
-   :start-after: fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_start
-   :end-before: fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_end
+   .. tab:: MCUboot
 
-.. note::
-   If the :kconfig:option:`CONFIG_MCUMGR_GRP_IMG_REJECT_DIRECT_XIP_MISMATCHED_SLOT` Kconfig option is enabled in the application configuration, the device rejects update image upload for the invalid slot.
-   It is recommended to enable the option if the application uses MCUboot in the direct-xip mode.
+      .. include:: /device_guides/nrf52/fota_update.rst
+         :start-after: fota_upgrades_over_ble_nrfcdm_common_dfu_steps_start
+         :end-before: fota_upgrades_over_ble_nrfcdm_common_dfu_steps_end
+
+      .. include:: /device_guides/nrf52/fota_update.rst
+         :start-after: fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_start
+         :end-before: fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_end
+
+      .. note::
+         When the :kconfig:option:`CONFIG_MCUMGR_GRP_IMG_REJECT_DIRECT_XIP_MISMATCHED_SLOT` Kconfig option is enabled in the application configuration, the device rejects the update image upload for the invalid slot.
+         It is recommended to enable the option if the application uses MCUboot in the direct-xip mode.
+
+   .. tab:: SUIT
+
+      1. Generate the SUIT envelope by building your application with the FOTA support over Bluetooth Low Energy.
+         You can find the generated :file:`root.suit` envelope in the :file:`<build_dir>/DFU` directory.
+      #. Download the :file:`root.suit` envelope to your device.
+
+         .. note::
+            `nRF Connect for Desktop`_ does not currently support the FOTA process.
+
+      #. Use the `nRF Connect Device Manager`_ mobile app to update your device with the new firmware.
+
+         a. Ensure that you can access the :file:`root.suit` envelope from your phone or tablet.
+         #. In the mobile app, scan and select the device to update.
+         #. Switch to the :guilabel:`Image` tab and tap on :guilabel:`ADVANCED` in the upper right corner of the app.
+         #. In the **Firmware Upload** section, tap the :guilabel:`SELECT FILE` button and select the :file:`root.suit` envelope.
+         #. Tap the :guilabel:`START` button.
+         #. Wait for the DFU to finish and then verify that the application works properly.
+
+      .. note::
+         Support for SUIT updates is available since the following versions of the `nRF Connect Device Manager`_ mobile app:
+
+         * Version ``1.10`` on Android.
+         * Version ``1.6`` on iOS.
```

Comment:
smp_transfer sample lists 1.9 for android and 1.6 for iOS.
Why we have different version here?

Reply:
I am pretty sure they haven't updated their doc but will check with Charon and post their answer here.

Reply:
As far as I know there was some kind of SUIT support already done for the 1.5.3 for iOS (https://github.com/NordicSemiconductor/IOS-nRF-Connect-Device-Manager/releases/tag/1.5.3) and 1.9 for Android (https://github.com/NordicSemiconductor/Android-nRF-Connect-Device-Manager/releases/tag/1.9.0), thus why I ask.

Reply:
Ok talked with Charon and using 1.9 doesn't work. Problem is that 1.10 is not yet released. but we will need it anyway to make it working - so I would leave it as it is.